### PR TITLE
Adding Comments and Comment-Permissions

### PR DESCRIPTION
**Pull request by stian8**
Issue #136 
Commenting is allowed under these permissions:
- Publicly editable
- Publicly commentable
- Private edit access
- Private comment access

Commenting works for original comments and summarized comments but not summary nodes.
You can see this in action through the "Reply" button.

### CODE REVIEW

File: wikum/private_config.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 MYSQL_PROD = {
```
Comment (amyxzhang) [inline wikum/private_config.py]: i'm guessing you do not want to commit changes to this file? :)

Comment (stian8) [reply wikum/private_config.py]: Oh yes, thanks for catching that! I've reverted this file now.

**Comment by amyxzhang** [comment]
@stian8 wait we don't actually want to *delete* private_config.py. It's meant to be in the repo as a template that you then edit in your local environment.

**Comment by stian8** [comment]
Got it, sorry about that. It should be re-added now.

**Pull request merged**